src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/errorsummary/ErrorSummary.scala

```diff
@@ -26,7 +26,7 @@ case class ErrorSummary(
   attributes: Map[String, String] = Map.empty,
   title: Content = Empty,
   description: Content = Empty,
-  disableAutoFocus: Boolean = false
+  disableAutoFocus: Option[Boolean] = Some(false)
```

Comment:
@kyle-bowden just realising with my Friday brain that where existing params have changed from Boolean to Option[Boolean] is a breaking change, wonder if we should go back to what we had before and just have a slight discrepancy with govuk API for backwards compatibility since I checked and [default is still false in the javascript](https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/components/error-summary/error-summary.mjs#L33) - or if we should do a major version bump and describe the changes needed changelog